interview.md

```diff
@@ -11,41 +11,94 @@ _What need is met by your idea? Who are you helping? What is that person's
+about why he and his teammates found the complication aspect of the molecule
+visualization programs confusing.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Interpreted molecule visualization programming language.
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+The user would be able to write in a coding language where they would define
+a molecules by defining atoms, bonds and the connections between the atoms and
+bonds. Then they could run the program and a visualization of the molecule would
+be created. Additionally, there could be information given about the molecule
+(maybe polarity, bond strength, etc.) after it is created. 
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When the program runs it will either have errors or create a visualization of
+the molecule. There can only be runtime errors, and these would include if the
+molecule was not defined correctly (issue with bonds and atoms connections not
+matching up). They would be communicated to the user through error messages.
+
+OR, there might be a way in which we could still visualize errors with the
+creation of the molecule. Let's say an atom needs an additional bond, there
+might be a way to show in the visualization that the atom is missing a bond and
+tell the user that they should fix this. That way it would be easier to debug.
+Then, there would still be runtime syntax errors.
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to create visualizations of molecules and get basic information
+about the molecules. It should be difficult to create molecules that can't
+actually exist according to chemistry first principles.
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+Yes, there are programs that allow you to create molecules, we even used one
+in the frosh core chemistry class. The type of programs that already exist
+are very large, and as such can get very complicated and more difficult to use.
+
+The project that we used in frosh chem is called crystal maker. While this one
+is specifically focused on creating full crystals, it is a very large and
+complicated program to use.
+https://crystalmaker.com/
+
+What is unique about this project, is that it is very focused, and should then
+be a much simpler and more intuitve program for chemists to use.
+
```

Comment:
I like the idea of narrowing down the scope such that it just does one thing, but it does that one thing so well it makes up for the lack of flexibility in function. At the same time, it might be worth asking the chemists what additional features they might deem as absolutely  necessary before reducing the program too much.